docs/README.md

```diff
@@ -21,6 +21,8 @@ Perform the actions from the sub-documents in the following order:
 
```

Comment:
Just adding a comment here to make sure we capture this work in a Jira somewhere.  The pull_openstack_configuration.md file discusses the use of os-diff to capture config differences for different services.  We are going to need to update that tool to add code for barbican and keystone.  In particular, see https://github.com/openstack-k8s-operators/os-diff/tree/main#add-service

Reply:
Resolve this comment once the relevant Jira are created.

Reply:
In stop_openstack_services.md, there is a script that is used to stop openstack services.  Does it need an addition to stop barbican too?

Reply:
I added barbican to the list of stopped services